src/components/SearchBar/icons/filter-lines.svg

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="iso-8859-1"?>

```

Comment:
Instead of using an actual svg, you can make this a component that returns an svg:

```jsx
class FilterLines extends Component {
  render() {
    return (
      <svg>
        // rest of svg data
      </svg>
    );
  }
}
```